libuv/plan.sh

```diff
@@ -0,0 +1,45 @@
+pkg_name=libuv
+pkg_origin=core
+pkg_version="1.16.1"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source="http://dist.$pkg_name.org/dist/v$pkg_version/$pkg_name-v$pkg_version.tar.gz"
```

Comment:
it would be good to get all these variables enclosed in brackets (and the line below too).